### PR TITLE
rtv: fix update to version 1.19.0

### PR DESCRIPTION
**Pull request by jwpincus**
-new version that handles reddit api change
-installs version 1.19.0
-tested locally

- [ x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/rtv.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ class Rtv < Formula
   depends_on :python3
 
   def install
-    venv = virtualenv_create(libexec, "python3")
+    venv = virtualenv_create(libexec, "python")
```
Comment (ilovezfs) [inline Formula/rtv.rb]: that needs to be python3.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/rtv.rb (hunk 1 above)
Comment (jwpincus) [reply Formula/rtv.rb]: It wouldn't compile locally for me with python3. Happy to change it, but I can't promise that it'll work.

**Comment by ilovezfs** [comment]
@jwpincus we virtually never switch something back to python2 after it has been switched to python3. Please put it back the way it was. If it fails, we'll need to figure out why.

**Comment by jwpincus** [comment]
@ilovezfs Set it back to use python3.

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @jwpincus! Without people like you submitting PRs we couldn't run this project. You rock!

**Review by ilovezfs — APPROVED** [review]

**Pull request closed without merging**